# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gnomemm12/gcc33-fix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
This patch is extracted from
http://mail.gnome.org/archives/gtkmm-list/2003-September/msg00016.html


--- gnomemm-1.2.4.orig/src/gnome--/app-helper.h
+++ gnomemm-1.2.4/src/gnome--/app-helper.h
@@ -261,6 +261,7 @@
     { return static_cast<T_Info&>(begin_[n]); }
 
   GnomeUIInfo* gtkobj() const 
-    { return static_cast<GnomeUIInfo*>(data_); }
+    //{ return static_cast<GnomeUIInfo*>(data_); }
+    { return (GnomeUIInfo*)(data_); }
 
   vector<Info> make_return_vector(); //Used by get_uitree().
   vector<SubTree> make_return_vector_subtrees(); //Used by Gnome::App.
